### ISSUE TITLE
Missing a Delete Domain Command

### ISSUE THREAD

**Issue by tibord** [devel/PowerDNS-API#7]
Is there an API command to delete the domain and all its records? or at least a command that will delete the domain after all the records have been deleted with the DELETE /api/record/example.com/123 command

**Comment by abh** [comment]
Yeah, I didn't add that.  It seemed like such a dangerous command to have.  ;-)

I like the implicit idea you have of requiring all records to be deleted first (maybe absent some sort of "no, really – I want it gone" parameter).  What do you think?

**Comment by tibord** [comment]
Yes that would be cool. Because if a domain has all the records deleted and somebody tries to delete the actual domain that would be a clear indication that somebody wants it gone. Also it would allow the PowerDNS-API to have a full set of commands for working with PowerDns as it is now you would have to clear out the orphan domains from the db directly, because you can only delete the records of it by using the API.

**Comment by christecc** [comment]
A delete domain command would be cool.

**Comment by 00000SZ** [comment]
+1
